internal/search/streaming/search_filters.go

```diff
@@ -107,6 +158,33 @@ func (s *SearchFilters) Update(event SearchEvent) {
 		}
 	}
 
+	addSymbolFilter := func(symbols []*result.SymbolMatch, limitHit bool) {
+		for _, sym := range symbols {
+			selectKind := result.ToSelectKind[strings.ToLower(sym.Symbol.Kind)]
+			filter := fmt.Sprintf(`select:symbol.%s`, selectKind)
+			s.filters.Add(filter, selectKind, 1, limitHit, "symbol type")
+		}
+	}
+
+	addCommitAuthorFilter := func(commit gitdomain.Commit) {
+		filter := fmt.Sprintf(`author:%s`, commit.Author.Email)
+		s.filters.Add(filter, commit.Author.Name, 1, false, "author")
+	}
+
+	addCommitDateFilter := func(commit gitdomain.Commit) {
+		var cd time.Time
+
+		if commit.Committer != nil {
```

Comment:
super tiny style nit: when variables are declared then initialized in an `if` branch, I find it easier to read if it's grouped with the initializer block.
```suggestion
		var cd time.Time
		if commit.Committer != nil {
```